client/signup/utils.js

```diff
@@ -11,15 +11,39 @@ import { translate } from 'i18n-calypso';
 import steps from 'signup/config/steps-pure';
 import flows from 'signup/config/flows';
 import userFactory from 'lib/user';
+import { abtest } from 'lib/abtest';
 
 const user = userFactory();
 
 const { defaultFlowName } = flows;
 
+function isEligibleForSwapStepsTest() {
+	const cookies = cookie.parse( document.cookie );
+	const countryCodeFromCookie = cookies.country_code;
+	const isUserFromUS = 'US' === countryCodeFromCookie;
```

Comment:
It looks like the abtest library might already have a way to limit the test to a particular country using the `countryCodeTargets` property described in the readme: https://github.com/Automattic/wp-calypso/blob/master/client/lib/abtest/README.md